<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1916</width>
    <height>1025</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>901</width>
      <height>185</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="trainSaveFileLocation" stretch="1,0,1,0,0,0,0,0">
     <item>
      <widget class="QLabel" name="labelFileDir">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>파일 위치</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Train</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="trainOk" stretch="1,5,1">
       <item>
        <widget class="QLabel" name="labelTrainOkTitle">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTrainOkDir">
         <property name="text">
          <string>이미지 데이터 경로를 추가해주세요</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTrainOkCount">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="trainDef" stretch="1,5,1">
       <item>
        <widget class="QLabel" name="labelTrainDefTitle">
         <property name="text">
          <string>Def</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTrainDefDir">
         <property name="text">
          <string>이미지 데이터 경로를 추가해주세요</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTrainDefCount">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="TestRatio" stretch="1,5,1">
       <item>
        <widget class="QLabel" name="labelTestRatioTitle">
         <property name="text">
          <string>비율 :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTestRatioDir">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTestRatioCount">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="TestDef" stretch="1,5,1">
       <item>
        <widget class="QLabel" name="labelTestDefTitle">
         <property name="text">
          <string>Def</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTestDefDir">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTestDefCount">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="TestOk" stretch="1,5,1">
       <item>
        <widget class="QLabel" name="labelTestOkTitle">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTestOkDir">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTestOkCount">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>1801</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>table - recall precision</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>table1 - loss</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>table - acc</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="lblAreaLoss">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QLabel" name="lblAreaAcc">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2">
      <widget class="QLabel" name="lblAreaRecall">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>660</y>
      <width>1071</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutLog" stretch="1,5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
       <item>
        <widget class="QLabel" name="labelLog">
         <property name="text">
          <string>로그</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>log clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>940</y>
      <width>1841</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>660</y>
      <width>151</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
     <item>
      <widget class="QLabel" name="labelControl_2">
       <property name="text">
        <string>제어</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonControlStart">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>시작</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonControlRestart">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>다시시작</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonControlPause">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>일시정지</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonControlStop">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>정지</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>660</y>
      <width>341</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,1,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="labelParameter">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>163</height>
          </size>
         </property>
         <property name="text">
          <string>파라미터 설정</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
       <item>
        <widget class="QLabel" name="labelEpoch">
         <property name="text">
          <string>Epoch</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxEpoch">
         <property name="minimum">
          <number>25</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1,1">
       <item>
        <widget class="QLabel" name="labelBatchSize">
         <property name="text">
          <string>Batch Size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxBatchSize">
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>64</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>사용자 지정</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditBatchSize">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3,1">
       <item>
        <widget class="QLabel" name="labelLearningRateName">
         <property name="text">
          <string>Learning Rate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderLearningRate">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelLearningRate">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1,1">
       <item>
        <widget class="QLabel" name="labelDecayStep">
         <property name="text">
          <string>Decay Step</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxDecayStep">
         <item>
          <property name="text">
           <string>600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>사용자 지정</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditDecayStep">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="checkBoxAugmentation">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>680</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Agumentation</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>720</y>
      <width>91</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkBoxFlip">
       <property name="text">
        <string>Flip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxSpin">
       <property name="text">
        <string>Spin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxSwift">
       <property name="text">
        <string>Shift</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxMixup">
       <property name="text">
        <string>Mix up</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>0</y>
      <width>901</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="testValidationFileLocation" stretch="0,0,0,0,0">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Validation</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ValidationRatio" stretch="1,5,1">
       <item>
        <widget class="QLabel" name="labelValidationRatioTitle">
         <property name="text">
          <string>비율 :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelValidationRatioDir">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelValidationRatioCount">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ValidationOk" stretch="1,5,1">
       <item>
        <widget class="QLabel" name="labelValidationOkTitle">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelValidationOkDir">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelValidationOkCount">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ValidationDef" stretch="1,5,1">
       <item>
        <widget class="QLabel" name="labelValidationDefTitle">
         <property name="text">
          <string>Def</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelValidationDefDir">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelValidationDefCount">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="saveRoute" stretch="1,5,1">
       <item>
        <widget class="QLabel" name="labelSave">
         <property name="text">
          <string>model save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSaveDir">
         <property name="text">
          <string>경로</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1916</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_other_File">
   <property name="text">
    <string>Open other File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
